docs/config.md

```diff
@@ -33,6 +32,7 @@ Variables not exclusively used by gopass
 | `PAGER`                | `string` | the pager program used for `gopass list`. See [Features](features.md#auto-pager) for details           |
 | `GIT_AUTHOR_NAME`      | `string` | name of the author, used by the rcs backend to create a commit                                         |
 | `GIT_AUTHOR_EMAIL`     | `string` | email of the author, used by the rcs backend to create a commit                                        |
+| `NO_COLOR`             | `bool`   | disable color output
```

Comment:
I would mention the https://no-color.org/ thing, because setting a global NO_COLOR env can have unexpected side effects on their system if people are not aware it's a "common" variable.